spec/unit/puppet-lint/checks_spec.rb

```diff
@@ -53,7 +53,7 @@
         end
 
         it 'creates a syntax error problem for the file' do
-          expect(instance.problems).to have(1).problem
+          expect(instance.problems.size).to eq(1)
           expect(instance.problems.first).to include(
```

Comment:
While writing this PR I realized you can replace this with `contain_exactly` but didn't want to go back and rewrite everything.
```suggestion
          expect(instance.problems).to contain_exactly(
            include(
```